youtube_dl/extractor/vlive.py

```diff
@@ -34,26 +35,44 @@ def _real_extract(self, url):
         webpage = self._download_webpage(
             'http://www.vlive.tv/video/%s' % video_id, video_id)
 
-        status = self._download_json(
+        # UTC+x - UTC+9 (KST)
+        tz = time.altzone if time.localtime().tm_isdst == 1 else time.timezone
+        tz_offset = -tz // 60 - 9 * 60
+        status_params = self._download_json(
             'http://www.vlive.tv/video/status?videoSeq=%s' % video_id,
             video_id, 'Downloading JSON status',
-            headers={'Referer': url}).get('status')
+            headers={'Referer': url, 'Cookie': 'timezoneOffset=%d' % tz_offset})
```

Comment:
Use ```self._add_cookies``` to add global cookies. It's better to use the same timezone offset for all requests to get consistent results.